apps/mcp-ui-app/package.json

```diff
@@ -0,0 +1,24 @@
+{
+  "name": "@lg-apps/mcp-ui-app",
+  "version": "0.1.0",
+  "private": true,
+  "scripts": {
+    "dev": "next dev",
+    "build": "next build",
```

Comment:
Turbo is complaining that there's nothing to cache when I run this. Probable need to add a custom turbo config for the app that sets the outputs https://turborepo.com/docs/reference/configuration#outputs
```
    ...Finishing writing to cache...                                                                     
WARNING  no output files found for task @lg-apps/mcp-ui-app#build. Please check your `outputs` key in `turbo.json`
```